HW-09/homework09.py

```diff
@@ -0,0 +1,17 @@
+num = input("Number Fibonacci: ")
+num = int(num)
+
+def my_gen(num):
+    fibonacci1 = 1
+    fibonacci2 = 1
+    item = 0
+    while item < num - 2:
+        add_fibonacci = fibonacci1 + fibonacci2
+        fibonacci1 = fibonacci2
+        fibonacci2 = add_fibonacci
+        item = item + 1
+        yield f'Element value {fibonacci2}'
+
+for value in my_gen(num):
+    print(value)
```

Comment:
You should print only the element of the num position in the sequence

Reply:
The fix for this comment should be something related to this code. You shouldn't have changed the generator. Rollback the generator changes.
To make it work you can, for example, transform your generator to a list and print its last value: `print(list(my_gen(num))[-1])`